java/src/jmri/managers/AbstractLightManager.java

```diff
@@ -118,7 +118,9 @@ public Light newLight(@Nonnull String systemName, @CheckForNull String userName)
      * @param systemName the system name to use for this light
      * @param userName   the user name to use for this light
      * @return the new light or null if unsuccessful
+     * @throws IllegalArgumentException something failed in the names
      */
+    @SuppressWarnings("javadoc")
```

Comment:
Why is javadoc warning suppressed?

Reply:
I suspect it was something automatic by Eclipse while adding the throws.

Reply:
If I was to make those two changes, do I reopen this PR or what is the suggested method?

Reply:
- Create a new branch from upstream/master. This is the default for GitHub Desktop.
- Add the changes to that branch.
- Then create a new PR.